src/content/docs/en/guides/routing.mdx

```diff
@@ -296,6 +296,32 @@ if (!isLoggedIn(cookie)) {
 </html>
 ```
 
+## Rewrites
+
+<Since v="4.13.0" />
+
+A rewrite allows you serve a different route without redirecting the browser to a different page. The browser will show the same address in the URL bar, but will display the content of the URL provided to `Astro.rewrite()`.
+
+This can be useful for showing the same content at multiple paths (e.g. `/products/shoes/men/` and `/products/men/shoes`/).
+
+For content that has permanently moved, or to direct your user to a different page (e.g. a user dashboard after logging in), use a [redirect](#redirects) instead.
+
+A rewrite is really useful in an i18n context, for example. Let's suppose your website serves localised version of Spanish e.g. `/es-es/articles/introduction` (Spain) and `/es-mx/articles/introduction` (Mexico). However, your website also serves pages for `es-CU` (Cuba), but you're missing the localised content for `/es-cu/articles/introduction`. 
+
+However, you don't to serve that page as a 404, and you don't want to serve a redirect for SEO purpose. That's where the rewrites comes into play. You can decide to **fallback** to the Mexican localised page using the new `Astro.redirect` method:
```

Comment:
```suggestion
Rewrites are also useful to provide fallback content, such as for a localized page route whose content does not exist. If you don't want to serve a redirect for SEO purposes, and do not want to show a 404, then you can use a rewrite to display a different page at the requested route. The following example uses a rewrite to display the `/es-mx/` version of a page (Mexican Spanish) without from the `/es-CU/` (Cuban Spanish) URL path. 
```

Just rough draft for now, but I'm thinking all of this could be condensed to something like this...